.gitignore

```diff
@@ -1,5 +1,4 @@
 *temp*
-assets/
```

Comment:
Ah - had to look again. You're absolutely right, the hashes are only created from files that were pulled out of assets. So go ahead and just serve what's in assets straight out of memory.

We'll have to add the functionality I'm talking about in https://github.com/Mierdin/todd/issues/27. Sorry for the confusion.